src/main/index.js

```diff
@@ -12,6 +12,8 @@ if (process.env.NODE_ENV !== 'development') {
     .replace(/\\/g, '\\\\')
 }
 
+alert('foooo')
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Remove this usage of alert(...). ([javascript:S1442](https://sonarcloud.io/organizations/bellingard/rules#rule_key=javascript%3AS1442))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=my-account-manager&pullRequest=5&issues=AWJrSvaN-V-rHO1Cdi45&open=AWJrSvaN-V-rHO1Cdi45)

[ISSUE_KEY]: AWJrSvaN-V-rHO1Cdi45